src/darker/black_diff.py

```diff
@@ -35,13 +35,15 @@
 
 import logging
 import sys
+from distutils.version import LooseVersion
```

Comment:
I think #189 is a simpler fix here, but if you prefer doing it by version I wouldn’t use this class. Its comparison operators just return whatever if a version doesn’t parse right.

These days I always use [`packaging.version`](https://packaging.pypa.io/en/latest/version.html).

Reply:
`packaging` isn't currenly required by Darker, and it feels wrong to add that dependency just because of this compatibility hack. Your approach in #189 makes more sense.